app/amity.py

```diff
@@ -0,0 +1,309 @@
+import random
+import os
+
+from app.livingspace import LivingSpace
+from app.office import Office
+from models.amity_database import Room, Person as PersonModel, Session
+from app.people import Person
+
+
+session = Session().create_session()
+
+
+class Amity(object):
+
+    """Class Amity"""
+    rooms = []
+    persons = []
+    unallocated = []
+
+    def __init__(self, session):
+        """Initializes a session with the database, so that load_state()
+        loads data into the app, from the main database alloamity_db"""
+        self.session = session
+
+    def load_state(self, db_name='alloamity_db.sqlite'):
+        """Loads data into application from database."""
+        if os.path.exists('alloamity_db.sqlite'):
+            all_rooms = self.session.query(Room).all()
+            for list_all_rooms in all_rooms:
+                Amity.rooms.append(list_all_rooms)
+
+            all_persons = self.session.query(PersonModel).all()
+            for list_all_persons in all_persons:
+                Amity.persons.append(list_all_persons)
+
+            os.remove('alloamity_db.sqlite')
+            return 'Data has been loaded into AlloAmity.'
+
+        else:
+            print("Default database alloamity_db.sqlite doesn't exist.")
+            return "Default database doesn't exist."
+
+    def add_person(
+        self,
+        first_name,
+        last_name,
+        job_group,
+        want_accomodation='N'
+    ):
+        """Adds person into an office and living space randomly."""
+        if job_group.lower() not in ['staff', 'fellow']:
+            print(
+                'Invalid input for job_group. Use: FELLOW for fellow, or STAFF for staff.')
+            return 'Wrong input.'
+
+        if want_accomodation.lower() not in ['y', 'n', 'yes', 'no']:
+            print('Invalid input for want_accomodation. Use: Y for yes, or N for no.')
+            return 'Wrong input.'
```

Comment:
Use `True` or `False` in your return statement. The string are being returned and not used anywhere in your `amity_docopt.py` and the strings will make your testing efforts tedious since you have to provide all the same return messages for failures or successes.

Follow the example given as below in your methods.

```python
def add_person(
        self,
        first_name,
        last_name,
        job_group,
        want_accomodation='N'
    ):
        """Adds person into an office and living space randomly."""
        if job_group.lower() not in ['staff', 'fellow']:
            print('Invalid input for job_group. Use: FELLOW for fellow, or STAFF for staff.')
            return False

        if want_accomodation.lower() not in ['y', 'n', 'yes', 'no']:
            print('Invalid input for want_accomodation. Use: Y for yes, or N for no.')
            return False

        new_person = Person()
        new_person._id = str(len(Amity.persons) + 1)
        new_person.first_name = first_name
        new_person.last_name = last_name.upper()
        new_person.job_group = job_group
        new_person.want_accomodation = want_accomodation.upper()
        self.add_person_to_office(new_person)
        self.add_person_to_living_space(new_person)

        Amity.persons.append(new_person)
        return True
```

Then when testing it's as easy as:

```python
    def tests_if_amity_adds_person(self):
           ......
        # if Job Group is not in ['staff', 'fellow']
        self.assertEqual(self.amity.add_person('Dido', 'Manjik', 'Fellow', 'Y'), False)
```